cmd/consumer-audit/main.go

```
@@ -87,6 +92,57 @@ func main() {
+			fmt.Fprintf(os.Stderr, "consumer-audit: superseded report: %v\n", err)
+			os.Exit(1)
+		}
+		printSupersededReport(out, sup)
+
+		if *supersededEnforce {
+			// An incomplete index is checked first and reported separately:
+			// a clean usage result drawn from a partial index is not a pass,
+			// it is an audit that could not see the whole tree. The distinct
+			// exit code lets CI tell the two failures apart.
+			if !sup.IsComplete() {
+				fmt.Fprintln(out)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)